resources/benchmark-runner.mjs

```diff
@@ -116,6 +116,106 @@ class PageElement {
     }
 }
 
+class MeasureTask {
```

Comment:
I'd call this MeasureStep or StepRunner since we'd be running & measuring individual BenchmarkStep here.